froide_evidencecollection/models.py

```diff
@@ -9,111 +9,303 @@
 from froide.publicbody.models import PublicBody
 
 
-class Source(models.Model):
-    note = models.TextField()
-    url = models.URLField(unique=True)
-    document_number = models.TextField(blank=True)
-    public_body = models.ForeignKey(PublicBody, on_delete=models.PROTECT, null=True)
-
-    def clean(self):
-        if (self.public_body and not self.document_number) or (
-            not self.public_body and self.document_number
-        ):
-            raise ValidationError(
-                "Either both or neither of document_number and public_body must be set"
-            )
-        super().clean()
+class PersonOrOrganization(models.Model):
+    external_id = models.PositiveIntegerField(
+        unique=True, verbose_name=_("external ID")
```

Comment:
is there a reason to explicitly set the verbose name? otherwise i would suggest removing it if it's the same as the auto-generated verbose name (same with all other models)

```suggestion
        unique=True
```

Reply:
I explicitly set the verbose name to make it translatable, mostly for consistency. It’s not critical for this field since it’s not user-facing, but it may show up in the admin.